### PR TITLE
Modify gcp-crio vars to support cri-o runtime deployment

### PR DESCRIPTION
**Pull request by mtnbikenc**
Limited alternative to #1307

To support https://github.com/openshift/openshift-ansible/pull/9732

### CODE REVIEW

**Comment by sdodson** [comment]
@mrunalp @smarterclayton FYI, Russ has other work in progress to fix issues where a cluster is mixed docker and cri-o and he ran into this which leads us to believe that while we may be installing cri-o we're not actually configuring the kubelet to use it.

**Comment by stevekuznetsov** [comment]
@runcom @mrunalp

**Comment by mrunalp** [comment]
Looks fine. I will test it out.

File: cluster/test-deploy/gcp-crio/vars.yaml [hunk 1]
```diff
@@ -32,13 +31,40 @@ openshift_node_groups:
   - 'node-role.kubernetes.io/master=true'
   - 'node-role.kubernetes.io/infra=true'
   - 'role=infra'
+  - 'runtime=cri-o'
+  edits:
```
Comment (smarterclayton) [inline cluster/test-deploy/gcp-crio/vars.yaml]: These are all settings that should be part of the role, not in the config.

**Comment by sdodson** [comment]
> Why are we changing the config and not the role?

If oo_first_master where we create the configmaps isn't openshift_use_crio=true then you have to push this into node group config.

**Comment by sdodson** [comment]
For anyone not following the linked PR, there's more discussion in https://github.com/openshift/openshift-ansible/pull/9732#issuecomment-417059175

**Comment by mtnbikenc** [comment]
Closing until we have further direction on mixed cri-o and docker deployments.

**Pull request closed without merging**